packages/flutter/lib/src/animation/tween.dart

```diff
@@ -99,7 +107,10 @@ class RectTween extends Tween<Rect> {
   Rect lerp(double t) => Rect.lerp(begin, end, t);
 }
 
-/// An animated variable containing a int.
+/// An interpolation between two integers.
+///
+/// This class specializes the interpolation of Tween<int> to be
+/// appropriate for integers.
```

Comment:
"...by interpolating between the given begin and end values and then rounding the result to the nearest integer."